# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/library/rust/shared/Cargo.toml
# Commit: 63b555e1f503
# Full Hash: 63b555e1f503db91a54604fd8cef88dfab7c0c7a
# Author: Dana Keeler <dkeeler@mozilla.com>
# Date: 2023-06-08 09:27:30
# Regressor Bug: 1834862
# File Overlap Count: 1
# Description:
#   Bug 1834862 - implement nsICryptoHash in rust r=jschanck,supply-chain-reviewers
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D179727
# ==============================================================================

diff -r 6337ce701771 -r 63b555e1f503 toolkit/library/rust/shared/Cargo.toml
--- a/toolkit/library/rust/shared/Cargo.toml	Wed Jun 07 22:44:15 2023 +0000
+++ b/toolkit/library/rust/shared/Cargo.toml	Wed Jun 07 23:03:21 2023 +0000
@@ -38,6 +38,7 @@
 jsrust_shared = { path = "../../../../js/src/rust/shared" }
 cascade_bloom_filter = { path = "../../../components/cascade_bloom_filter" }
 cert_storage = { path = "../../../../security/manager/ssl/cert_storage" }
+crypto_hash = { path = "../../../../security/manager/ssl/crypto_hash" }
 bitsdownload = { path = "../../../components/bitsdownload", optional = true }
 storage = { path = "../../../../storage/rust" }
 bookmark_sync = { path = "../../../components/places/bookmark_sync", optional = true }